### PR TITLE
Cascade deletes to main.tsearch

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: commons/c2cgeoportal_commons/alembic/env.py [hunk 1]
```diff
@@ -50,6 +50,25 @@ def get_config():
...
+        if type_ == 'table':
+            return object.schema == _schema
+        else:
+            return object.table.schema == _schema
+    if _schema:
+        settings.update({
+            'target_metadata': Base.metadata,
+            'include_schemas': True,
+            'include_object': include_object
+        })
```
Comment (sbrunner) [inline commons/c2cgeoportal_commons/alembic/env.py]: For what is this code for?

File: commons/c2cgeoportal_commons/alembic/main/e85afd327ab3_cascade_deletes_to_tsearch.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+
+def downgrade():
+    schema = config['schema']
+
+    op.drop_constraint('tsearch_role_id_fkey', 'tsearch', schema='main', type_='foreignkey')
+    op.create_foreign_key('tsearch_role_id_fkey', 'tsearch', 'role', ['role_id'], ['id'], source_schema='main', referent_schema='main')
+
+    op.drop_constraint('tsearch_interface_id_fkey', 'tsearch', schema='main', type_='foreignkey')
+    op.create_foreign_key('tsearch_interface_id_fkey', 'tsearch', 'interface', ['interface_id'], ['id'], source_schema='main', referent_schema='main')
```
Comment (sbrunner) [inline commons/c2cgeoportal_commons/alembic/main/e85afd327ab3_cascade_deletes_to_tsearch.py]: All the `'main'` should be replaced by schema ...

File: commons/c2cgeoportal_commons/alembic/env.py (hunk 1 above)
Comment (arnaud-morvan) [reply commons/c2cgeoportal_commons/alembic/env.py]: As the comment says, this is the autogenerate config, for details see:
https://alembic.sqlalchemy.org/en/latest/autogenerate.html
https://alembic.sqlalchemy.org/en/latest/api/runtime.html#alembic.runtime.environment.EnvironmentContext.configure.params.target_metadata

Comment (sbrunner) [reply commons/c2cgeoportal_commons/alembic/env.py]: OK, thanks :-)

File: commons/c2cgeoportal_commons/alembic/main/e85afd327ab3_cascade_deletes_to_tsearch.py (hunk 2 above)
Comment (arnaud-morvan) [reply commons/c2cgeoportal_commons/alembic/main/e85afd327ab3_cascade_deletes_to_tsearch.py]: Done, everything is green except the review.

**Comment by sbrunner** [comment]
Don't take care on Codacy review ...

**Pull request merged**